.drone.yml

```diff
@@ -59,6 +59,8 @@ pipeline:
     image: golang:1.10 # this step is kept as the lowest version of golang that we support
     pull: true
     commands:
+      - go get
```

Comment:
go get shouldn't be needed because necessary packages should be in vendor directory already.

Reply:
Except that go1.10 doesn’t support go.mod and so -mod=vendor won’t work and so it’ll try to fetch from cache instead.

Reply:
And since the go generate needs the cache to exist, if go1.10 is going to be supported, go get must come before go generate. Otherwise he generate can’t use go run, which would mean that any tool used by go generate would individually need its own go run before go generate could be called. Hence, I say we should drop go1.10 and start preparing for go1.13 instead.

Reply:
If you don’t understand that, just try the change yourself and read up on go.mod.

Reply:
@solderjs you can replace commande like  `go run github.com/jteeuwen/go-bindata/go-bindata` by `go run ./vendor/github.com/jteeuwen/go-bindata/go-bindata` and this go get should not be need.

Reply:
That’s not compatible with go.mod. In this particular instance it may _happen_ to work because there’s only one file to run, but in general it’s a bad pattern with inconsistent results.

Lets prepare for go1.13 rather than doing things in hacky ways that become more technical debt.

Reply:
When we drop go1.10 we can drop the go get and replace it with -mod vendor.

Reply:
@solderjs If I understand correctly, Go pre-1.10 still recognize the `vendor` directory. Once it is set up (that is, dependencies are pulled in), Go 1.10 will look it up anyway.  

It's that Go 1.10 cannot utilize go.mod to manage dependencies. But that's already the case sinec started using modules.

Reply:
Yes, we want the CI step to fail if deps aren't met, as like @typeless mentioned, go before modules was implemented will check the vendor dir. If this CI step fails it indicates that the vendor dir was not appropriately updated.